tests/src/e2e/ledger_tests.rs

```diff
@@ -2095,7 +2095,7 @@ fn proposal_submission() -> Result<()> {
     let mut client = run!(test, Bin::Client, query_proposal, Some(15))?;
     client.exp_string("Proposal Id: 0")?;
     client.exp_string(
-        "passed with 120900.000000 yay votes and 0.000000 nay votes (0.%)",
+        "passed with 120000.000000 yay votes and 900.000000 nay votes (0.%)",
```

Comment:
why did this change? :thinking:

Reply:
In test we submit two votes, a `yay` with the validator (120000 voting power) and a `nay` with a delegator (900 voting power) and previously we were not seeing the nay votes before because of a bung in the `is_same_side` function. I've tagged you at the wrong line

Reply:
ah